include/blmc_robots/solo.hpp

```diff
@@ -234,6 +234,7 @@ class Solo
   Vector8d joint_gear_ratios_; /**< joint gear ratios (9). */
   Vector8d motor_max_current_; /**< Max appliable current before the robot shutdown. */
   Vector8d joint_zero_positions_; /**< Offset to the theoretical "0" pose. */
+  Eigen::Array<double, 8, 1> max_joint_torques_; /**< Max joint torques (N/m) */
```

Comment:
Why do you use an `Eigen::Array` instead of `Eigen::Vector` here?

Reply:
See answer below and https://eigen.tuxfamily.org/dox/group__TutorialArrayClass.html

Reply:
In terms of using the same types, I would rather use a `Vector8D` here and then convert via `max_joint_torques_.array()` below. Not having strong opinions on this, so marking as resolved.